modules/gke-cluster/main.tf

```diff
@@ -292,6 +292,13 @@ resource "google_container_cluster" "cluster" {
     }
   }
 
+  dynamic "binary_authorization" {
+    for_each = var.enable_binary_authorization == true ? [""] : []
```

Comment:
the `== true` is redundant. Can you make the condition just `var.enable_binary_authorization`?

Reply:
Sure. I have removed ==true